pilot/test/mock/config.go

```diff
@@ -151,6 +151,8 @@ func Make(namespace string, i int) model.Config {
 	return model.Config{
 		ConfigMeta: model.ConfigMeta{
 			Type:      model.MockConfig.Type,
+			Group:     "config.istio.io",
```

Comment:
same, let's have at least the same 2 constants used instead of hardcoded strings

Reply:
For test, it doesn't really matter. It can be anything. It doesn't have to be config. 

After next PR, I will change it and make it more interesting...